android/authentication.md

```diff
@@ -1,615 +1,704 @@
-# Authentication
```

Comment:
What is the difference between here and #115 ?

Reply:
Nothing just the same fork